pangeo_forge_esgf/recipe_inputs.py

```diff
@@ -160,8 +164,15 @@ async def _esgf_api_request(
         content_type="text/json"
     )  # https://stackoverflow.com/questions/48840378/python-attempt-to-decode-json-with-unexpected-mimetype
     resp_data = resp_data["response"]["docs"]
+
     if len(resp_data) == 0:
         raise ValueError(f"No Files were found for {iid}")
+    # Since we have hacked CORDEX special case above, i'll do it here again:
+    # rename to common CMIP vocabulary if neccessary
+    resp_data = [
+        {"frequency" if k == "time_frequency" else k: v for k, v in r.items()}
+        for r in resp_data
+    ]
     return resp_data
```

Comment:
Sorry, this was neccessary since frequency facets have different names in CMIP6 and CMIP5 and CORDEX. I thinks this could really be tackled more easily in an object orient approach.